CLUEstering/CLUEstering.py

```diff
@@ -47,53 +45,47 @@ def test_blobs(n_samples: int, n_dim: int , n_blobs: int = 4, mean: float = 0,
         DataFrame containing n_blobs gaussian blobs.
     """
 
-    try:
-        if x_max < 0. or y_max < 0.:
-            raise ValueError('Error: wrong parameter value\nx_max and y_max must be positive.')
-        if n_blobs < 0:
-            raise ValueError('Error: wrong parameter value\nThe number of blobs must be positive.')
-        if mean < 0. or sigma < 0.:
-            raise ValueError("Error: wrong parameter value\nThe mean and sigma of the blobs"
-                             + " cannot be negative.")
-
-        centers = []
-        if n_dim == 2:
-            data = {'x0': [], 'x1': [], 'weight': []}
-            for i in range(n_blobs):
-                centers.append([x_max * rnd.random(),
-                                y_max * rnd.random()])
-            blob_data = make_blobs(n_samples=n_samples, centers=np.array(centers))[0]
-            for i in range(n_samples):
-                data['x0'] += [blob_data[i][0]]
-                data['x1'] += [blob_data[i][1]]
-                data['weight'] += [1]
-
-            return pd.DataFrame(data)
-        if n_dim == 3:
-            data = {'x0': [], 'x1': [], 'x2': [], 'weight': []}
-            sqrt_samples = int(sqrt(n_samples))
-            z_values = np.random.normal(mean,sigma,sqrt_samples)
-            for i in range(n_blobs):
-                centers.append([x_max * rnd.random(),  # the centers are 2D because we
-                                y_max * rnd.random()]) # create them for each layer
-            for value in z_values: # for every z value, a layer is generated.
-                blob_data = make_blobs(n_samples=sqrt_samples, centers=np.array(centers))[0]
-                for i in range(sqrt_samples):
-                    data['x0'] += [blob_data[i][0]]
-                    data['x1'] += [blob_data[i][1]]
-                    data['x2'] += [value]
-                    data['weight'] += [1]
-
-            return pd.DataFrame(data)
-
-        # If it gets to the bottom we raise the exception
-        raise ValueError("Error: wrong number of dimensions\nBlobs can only be generated"
+    if x_max < 0. or y_max < 0.:
+        raise ValueError('Wrong parameter value. x_max and y_max must be positive.')
+    if n_blobs < 0:
+        raise ValueError('Wrong parameter value. The number of blobs must be positive.')
+    if mean < 0. or sigma < 0.:
```

Comment:
maybe i am missing something, but why can't the `mean` be negative?

Reply:
You're right